R/tm_variable_browser.R

```diff
@@ -1335,3 +1339,17 @@ remove_outliers_from <- function(var, outlier_definition) {
   iqr <- q1_q3[2] - q1_q3[1]
   var[var >= q1_q3[1] - outlier_definition * iqr & var <= q1_q3[2] + outlier_definition * iqr]
 }
+
+#' Imputes empty strings as `NA`
+#'
+#' @param var (`vector`) a vector of any type and length
+#' @returns (`vector`) a vector with empty strings imputed as `NA`, if provided.
+#' @keywords internal
+impute_blanks_as_na <- function(var) {
```

Comment:
`teal.modules.general` `Imports` `tern`. Why write a new function rather than use one from a dependency?

Reply:
> `teal.modules.general` `Imports` `tern`. Why write a new function rather than use one from the dependency?

Reply:
I don't think there is such an imputation function in `tern`. However, It can be used when replacing `NA` with `<Missing>` 👍🏽

Reply:
Honestly, I don't think we should be even imputing things. The true bug is that the data had empty strings which should be the app developer's responsibility to fix before injecting it into the teal app.

Reply:
> I don't think there is such an imputation function in `tern`

Look again `?tern::df_explicit_na` :wink:

Reply:
Oh great! I'll use it. Thanks!